src/normalize/index.js

```diff
@@ -0,0 +1,129 @@
+const { SleepPlacement } = require('../enum')
+const { DEFAULT_OPTIONS } = require('../constants')
+const MIN_SLEEP = 500
+
+/**
+ * Validate archive without iterating nodes
+ * @param {HAR} archive
+ * @return {boolean}
+ */
+function isValidArchive(archive) {
+  return !!archive && !!archive.log && !!archive.log.entries
+}
+
+function getDraft(archive) {
+  const draft =
+    typeof archive === 'string'
+      ? JSON.parse(archive)
+      : JSON.parse(JSON.stringify(archive))
```

Comment:
I don't understand this bit. Is the intention to "validate" that the archive is in fact JSON compatible?

Reply:
Guess the idea was to support a JSON string. I will change it so that it requires a JSON object instead.